app/assets/stylesheets/components/_metadata-fields.scss

```diff
@@ -100,3 +100,13 @@
     width: 70%;
   }
 }
+
+.edit-subscription-form {
+  
+  .disabled {
```

Comment:
just want to make sure you verified this is consistent with where we may have other disabled fields